catalog/app/utils/APIConnector.js

```diff
@@ -276,19 +276,19 @@ function* apiSaga({ fetch, base = '', middleware = [] }) {
     mkFetchMiddleware({ fetch, base }),
   )
 
-  yield takeEvery(request.type, function* handleRequest({
-    payload: opts,
-    meta: { resolve, reject },
-  }) {
-    try {
-      const result = yield call(execRequest, opts)
-      yield put(response(result, opts))
-      yield call(resolve, result)
-    } catch (e) {
-      yield put(response(e, opts))
-      yield call(reject, e)
-    }
-  })
+  yield takeEvery(
+    request.type,
+    function* handleRequest({ payload: opts, meta: { resolve, reject } }) {
+      try {
+        const result = yield call(execRequest, opts)
+        yield put(response(result, opts))
+        yield call(resolve, result)
+      } catch (e) {
+        yield put(response(e, opts))
+        yield call(reject, e)
+      }
+    },
+  )
```

Comment:
I've called `npm run lint` and task have failed on this file with `prettier/prettier` error

Reply:
there was a newer version of prettier in my node_modules. fixed now